extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -337,8 +346,9 @@ export class AmpA4A extends AMP.BaseElement {
           // an acceptable solution to the 'Safari on iOS doesn't fetch
           // iframe src from cache' issue.  See
           // https://github.com/ampproject/amphtml/issues/5614
-          this.experimentalNonAmpCreativeRenderMethod_ =
-              fetchResponse.headers.get(RENDERING_TYPE_HEADER);
+          this.nonAmpCreativeRenderMethod_ =
+              fetchResponse.headers.get(RENDERING_TYPE_HEADER) ||
```

Comment:
Can't just `assert` because I don't want to fail the entire method.  Even if it gets an invalid value here, it will fall back to something that will _work_ during rendering.  But I agree that a dev notification is in order.  (Dev because the value of this header is in the control of the ad network, not the publisher.)  Added, along with a test for this case.